python/kfserving/kfserving/api/kf_serving_client.py

```diff
@@ -259,6 +276,9 @@ def rollout_canary(self, name, percent, namespace=None, # pylint:disable=too-man
         :return: inference service with canary endpoint
         """
 
+        if version != 'v1alpha2':
+            raise RuntimeError("The rollout_canary is only for KFServing v1alpha2.")
```

Comment:
Do we still need rollout API for v1beta1?

Reply:
No, no need for the v1beat1, but I think maybe some user still use old KFServing, but installed new KFS SDK, then the rollout still can work with old KFS, do you think no need to keep that? I can remove that if needed. Thanks. @yuzisun

Reply:
Just discussed with @yuzisun offiline, keep this now, and remove the API from next release. Thanks.